core/src/main/java/io/aiven/klaw/UiapiApplication.java

```diff
@@ -8,6 +9,7 @@
 import org.springframework.scheduling.annotation.EnableScheduling;
 
 @EnableScheduling
+@EnableSchedulerLock(defaultLockAtMostFor = "PT30S")
 @SpringBootApplication(
```

Comment:
I think we should make these settings configureable from the application.properties if we can.
It'll prevent people rebuilding for their own tried and tested settings

Reply:
Agree, a configuration would be good here, but couldn't find a way to configure it.